### PR TITLE
Add hotkey to toggle Docked/Handheld

### PR DESCRIPTION
**Pull request by HorrorTroll**
I don't know which key is best to set right now. So i'm set it at F1 key

### CODE REVIEW

File: Ryujinx.Common/Configuration/ConfigurationState.cs [hunk 1]
```diff
@@ -548,7 +549,7 @@ public void Load(ConfigurationFileFormat configurationFileFormat, string configu
                         },
                         Hotkeys        = new KeyboardHotkeys
                         {
-                            ToggleVsync = Key.Tab
+                            ToggleVsync  = Key.Tab
```
Comment (riperiperi) [inline Ryujinx.Common/Configuration/ConfigurationState.cs:552]: What is this aligning with?

Comment (riperiperi) [reply Ryujinx.Common/Configuration/ConfigurationState.cs:552]: Also, this specific instance doesn't set a docked key.

**Comment by HorrorTroll** [comment]
I don't know what to do for migrating, so to let hotkey work right now is open controller config and saved.

**Comment by HorrorTroll** [comment]
Closed until #1223 done

**Pull request closed without merging**